easybuild/easyconfigs/b/BCFtools/BCFtools-1.3.1-goolf-1.7.20.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'MakeCp'
+
+name = 'BCFtools'
+version = '1.3.1'
+
+homepage = 'http://www.htslib.org/'
+description = """BCFtools is a set of utilities that manipulate variant calls in the 
+ Variant Call Format (VCF) and its binary counterpart BCF"""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = ['https://github.com/samtools/bcftools/releases/download/%(version)s/']
+sources = [SOURCELOWER_TAR_BZ2]
+
+dependencies = [
+	('zlib', '1.2.8'),
+	('GSL', '1.16'),
+]
+
+buildopts = " USE_GPL=1 GSL_LIBS='-lgsl -lopenblas'"
```

Comment:
I am thinking about doing `buildopts = " USE_GPL=1 GSL_LIBS='-lgsl -l${LIBBLAS}'"` but checking the build log I see `LIBBLAS set to -lopenblas -lgfortran`

I am not sure if the `-lgfortran` can break something. opinions? @boegel does easybuild provides any env var which contains only `-lopenblas` ? I couldn't find it in the build log..